### DISCUSSION TITLE
Support for Alpine 3.13.1 / 3.13.2

### DISCUSSION THREAD

**Discussion by BogdanLivadariu** [dotnet/dotnet-docker#2679] [Q&A]
Based on the releases mentioned below, are there any official support builds for latest 3.1.13 runtime docker images ?

https://alpinelinux.org/releases/

**Comment by MichaelSimons** [comment]
The .NET images are tied to the floating <major.minor> Alpine image tags.  Whenever they are updated to point to the latest patch release, the .NET images get automatically rebuilt.  

The following snippet shows the 5.0 .NET Alpine floating tag is based on Alpine 3.13.2.  These steps should should the same for the other supported .NET versions (2.1 and 3.1)

``` bash
> docker pull mcr.microsoft.com/dotnet/runtime:5.0-alpine
5.0-alpine: Pulling from dotnet/runtime
ba3557a56b15: Already exists
71d959d5dea6: Pull complete
90ae4a24fd98: Pull complete
Digest: sha256:1a595c9f00679c38b58e1c8e033ca777f0cfc43461001c9c018c5b88079bd715
Status: Downloaded newer image for mcr.microsoft.com/dotnet/runtime:5.0-alpine
mcr.microsoft.com/dotnet/runtime:5.0-alpine
> docker run --rm mcr.microsoft.com/dotnet/runtime:5.0-alpine cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.13.2
PRETTY_NAME="Alpine Linux v3.13"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
```
